src/src/components/form/CustomInput.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+    <div :class="inputContainerClasses">
+        <!-- <label v-if="returnValue(prefix)" v-html="returnValue(prefix)"></label> -->
+        <fa v-if="input.fieldIcon" :icon="fieldIconChoice" class="input-icon"></fa>
+        <label :for="input.fieldName" class="input-label">
+        <input type="text"
+            :value="inputText"
+            @change="handleChange"
+            :class="customInputClasses"
+            :name="input.fieldName"
+            :id="id"
+            :required="input.fieldRequired"
+            :placeholder="input.fieldPlaceholder"
+            v-bind:disabled="input.disabled || input.readOnly">
+        </label>
+        <!-- <label v-if="returnValue(suffix)" v-html="returnValue(suffix)"></label> -->
```

Comment:
Saved for later?

Reply:
yes, I had to push these inbetween since this was an init branch. I'll resolve these in the next commit.